internal/sync.go

```diff
@@ -91,16 +91,33 @@ func (s *syncGSuite) SyncUsers() error {
 		ll.Debug("finding user")
 		uu, _ := s.aws.FindUserByEmail(u.PrimaryEmail)
 		if uu != nil {
-			s.users[uu.Username] = uu
```

Comment:
You can leave this, and just have the if statement. Because your else basically reflects this case.

```golang
s.users[uu.Username] = uu

if uu.Active == u.Suspended {
..
s.users[uu.Username] = upu
}
```
This is equal to your behavior and much easier to read.